src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -311,6 +311,9 @@
         - name: --max-count
           type: int
           short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 100]
+        - name: --uptime-sla
```

Comment:
I wonder what naming other services use that have a paid and free version. Wonder if we can be consistent in azure

Reply:
The naming is requested by PM, and agreed with several people who already has discussion about the naming. And the naming has been used for a while. It is not a good idea to change it.